### ISSUE TITLE
v2: fenced code blocks not being rendered correctly

### ISSUE THREAD

**Issue by grantpeltier** [russross/blackfriday#497]
Using v2, fenced code blocks are not getting parsed/rendered as expected. The `<code>` block is not getting wrapped with a `<pre>` and the language hint is not getting removed or used for the `<code>` class. Simple reproducible use case:

Input:
\``` javascript
var i = 1;
\```

Output:
\<p>\<code>javascript
var i = 1;
\</code>\</p>

**Comment by Mariusrik** [comment]
I have the same issue, it happens if you have a space behind the closing ```

**Comment by Mariusrik** [comment]
I have the same problem, it happens when you have a space behind the closing ```

**Comment by grantpeltier** [comment]
Good catch @Mariusrik! However, the issue for me was windows line endings. It appears this lib does not like `\r\n`.
